SampleApp/Warp-iosSampleApp.xcodeproj/xcshareddata/xcschemes/Finn.xcscheme

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1500"
```

Comment:
Are you using Xcode 16? Not sure if this will affect anything 🤔

Reply:
Yes I am using Xcode 16, Not sure, I will ask @milad-schibsted to help me make sure that it works with Xcode 15 too